README.md

```diff
@@ -0,0 +1,37 @@
+# TEMP_TITLE

```

Comment:
Think of a better title :P It may be changed later, of course. `WDI tester`, maybe?